tests/ci/pr_info.py

```diff
@@ -304,52 +304,60 @@ def has_changes_in_submodules(self):
         return False
 
     def can_skip_builds_and_use_version_from_master(self):
-        # TODO: See a broken loop
-        if FORCE_TESTS_LABEL in self.labels:
```

Comment:
I think we still need it. Logically, `FORCE_TESTS_LABEL` should start the full workflow

Reply:
ya makes sense to add it, if this function is called individually